src/widgets/index.js

```diff
@@ -15,4 +16,5 @@ export default {
   reasonList,
   dashboardOverview,
   topicFrequencyGraph,
+  targetPopulationTable,
```

Comment:
Not sure if you want to call this targetPopulationList to be in-line with reasonList (pref).